include/fixed_containers/concepts.hpp

```diff
@@ -117,6 +118,11 @@ concept ConstexprDefaultConstructible = requires() {
 template <class T>
 concept NotConstexprDefaultConstructible = not ConstexprDefaultConstructible<T>;
 
+template <typename T>
+concept ConstexprUnitConstructible = requires() {
+    { std::bool_constant<(T{std::monostate{}}, true)>() } -> std::same_as<std::true_type>;
+};
```

Comment:
Let's skip this and if a type is not default constructible, we can skip reflecting.